client/src/Ratings.jsx

```diff
@@ -1,59 +1,65 @@
+/* eslint-disable import/extensions */
 import React, { Component } from 'react';
-import PropTypes from 'prop-types';
+import axios from 'axios';
+
 import ReviewAction from './RatingsComponent/ReviewAction.jsx';
 import RatingsSearch from './RatingsComponent/RatingsSearch.jsx';
 import RatingsContent from './RatingsComponent/RatingsContent.jsx';
 import RatingsStarHeader from './RatingsComponent/RatingsStarHeader.jsx';
 import RatingsSummaryReview from './RatingsComponent/RatingsSummaryReview.jsx';
-const { data } = require('./RatingsComponent/RatingsDummyData.js');
-
-let newData = data.slice(0,2);
 
 class Ratings extends Component {
-  constructor(props) {
-    super(props);
+  constructor() {
+    super();
     this.state = {
-      ratings: newData,
+      ratingsStorage: [],
+      limitedRatings: [],
       ratingsCount: 2,
       showMoreRatings: true,
     };
     this.onAddMoreReviews = this.onAddMoreReviews.bind(this);
   }
 
-  // componentDidMount() {
-  //   axios.get('/api/reviews/?product_id=48487')
-  //     .then((results) => {
-  //       this.setState({
-  //         reviews: results.data.results,
-  //       });
-  //     })
-  //     .catch((err) => {
-  //       console.log(err);
-  //     });
-  // }
+  componentDidMount() {
+    this.getProductRatings();
+  }
 
   onAddMoreReviews() {
-    const { ratingsCount } = this.state;
+    const { ratingsStorage, ratingsCount } = this.state;
+    const ratingStorageLen = ratingsStorage.length;
 
     let newRatingsCount = ratingsCount + 2;
     let anyMoreRatings = true;
 
-    if (data.length < newRatingsCount) {
-      newRatingsCount = data.length;
+    if (ratingStorageLen < newRatingsCount) {
+      newRatingsCount = ratingStorageLen;
       anyMoreRatings = false;
     }
 
-    const newListings = data.slice(0, newRatingsCount);
+    const newListings = ratingsStorage.slice(0, newRatingsCount);
 
     this.setState({
-      ratings: newListings,
+      limitedRatings: newListings,
       ratingsCount: newRatingsCount,
       showMoreRatings: anyMoreRatings,
     });
-  };
+  }
+
+  getProductRatings() {
+    axios.get('/api/reviews/?product_id=48487')
+      .then((results) => {
+        this.setState({
+          ratingsStorage: results.data.results,
+          limitedRatings: results.data.results.slice(0, 2),
+        });
+      })
+      .catch((err) => {
+        throw err;
```

Comment:
suggestion: I think it would be more helpful for the team if this included more specific error handling so people know where it's coming from clearly and so everyone knows where to look for the error? Like a detailed console.log or a new Error

Reply:
Yea, we can create an error file that handles the different types of errors that we want to throw so that it is more uniformed.

Reply:
Some confusions. Not sure why the function is called getRatings when it gets reviews? Any reason? 
Also I would make it take the product id as an input for getReviews. In fact I would use the same inputs at all steps as the API allows in order to give the front end maximum flexibility. I would not hard code any routes. 
https://learn-2.galvanize.com/cohorts/2910/blocks/94/content_files/Front%20End%20Capstone/project-atelier-catwalk/reviews.md